static/js/stream_color.js

```diff
@@ -128,7 +128,7 @@ exports.sidebar_popover_colorpicker_options_full = {
     showInput: true,
     flat: true,
     cancelText: "",
-    chooseText: "choose",
+    chooseText: i18n.t("choose"),
```

Comment:
Hey @timabbott, this is what I was talking about. Opening the web app with this produces the `ReferenceError` I mentioned, because apparently `i18n` is out of scope here.